test/ja_serializer/dynamic_type_test.exs

```diff
@@ -22,24 +25,44 @@ defmodule JaSerializer.DynamicTypeTest do
   end
 
   test "dynamically assigns the type for single item" do
-    wilbur = JaSerializer.format(AnimalSerializer, @wilbur)
-    assert wilbur["data"]["type"] == "pig"
+    error_output =
+      capture_io(:stderr, fn ->
+        wilbur = JaSerializer.format(AnimalSerializer, @wilbur)
+        assert wilbur["data"]["type"] == "pig"
+      end)
+
+    error_output =~ @expected_error
```

Comment:
Should there be an assert here or is this intentional?

Reply:
Oh, I must have missed this one. Upon closer inspection, this line is also wrong. I definitely think there should be an assertion, but it should be `assert error_output == ""`.

I just fixed this in ba3a970.